bash_completion

```diff
@@ -2111,33 +2111,44 @@ _included_ssh_config_files()
     local configfile i files f
     configfile=$1
 
+    # From man ssh_config:
+    # "Files without absolute paths are assumed to be in ~/.ssh if included in a user configuration
+    # file or /etc/ssh if included from the system configuration file."
+    # This behavior is not affected by the the including file location -
+    # if system configuration file is included from user's config relative includes are still
+    # resolved in user's ssh config directory.
```

Comment:
```suggestion
    # This behavior is not affected by the including file location -
    # if the system configuration file is included from the user's config, relative includes are still
    # resolved in the user's ssh config directory.
```

Also, the codes, in particular comments, are supposed to be folded on column 80 so that the line has 79 characters at maximum.